.promu.yml

```diff
@@ -0,0 +1,25 @@
+go: 1.5.4
```

Comment:
Given that Prometheus and Alertmanager are now `1.6.2`, I think we can change here too.